tools/codegen/generate.bzl

```diff
@@ -19,7 +19,7 @@ MIXER_IMPORT_MAP = {
 # including the "../.." is an ugly workaround for differing exec ctx for bazel rules
 # depending on whether or not we are building within mixer proper or in a third-party repo
 # that depends on mixer proper. 
-MIXER_IMPORTS = [ "external/com_github_istio_api", "../../external/com_github_istio_api", "external/com_github_istio_mixer" ]
```

Comment:
unfortunately, we cannot do this. this is needed for external repo adapter integration.